app/helpers/merge_requests_helper.rb

```diff
@@ -0,0 +1,5 @@
+module MergeRequestsHelper
+  def display_value_or_placeholder(value, placeholder = "You didn't answer this question")
+    value.presence || content_tag(:span, placeholder, class: "app-!-colour-muted")
+  end
```

Comment:
Is there anywhere else we do this already that could  be reused?

Reply:
There are places with similar methods for questions and/or logs. Each not useful here. This method could be reusable though, doesn't have to just be for merge requests.

**check_answers_helper.rb**
```
def unanswered_value(log:, question:)
    if log.creation_method_bulk_upload? && log.bulk_upload.present? && !log.optional_fields.include?(question.id)
      "<span class=\"app-!-colour-red\">You still need to answer this question</span>".html_safe
    else
      "<span class=\"app-!-colour-muted\">You didn’t answer this question</span>".html_safe
    end
  end
```

**check_answers_summary_list_card_component.rb**
```
  def unanswered_value(question)
    if log.creation_method_bulk_upload? && log.bulk_upload.present? && !log.optional_fields.include?(question.id)
      "<span class=\"app-!-colour-red\">You still need to answer this question</span>".html_safe
    else
      "<span class=\"app-!-colour-muted\">You didn’t answer this question</span>".html_safe
    end
  end
```